pa11y/.pa11yci

```diff
@@ -24,6 +24,19 @@
       {
          "url": "https://digideps.local",
          "screenCapture": "screenshots/index.png"
+      },
+      {
+         "url": "https://digideps.local/report/{id}/overview",
+         "actions": [
+            "navigate to https://digideps.local/login",
+            "set field #login_email to behat-prof1@publicguardian.gov.uk",
+            "set field #login_password to Abcd1234",
+            "click element #login_login",
+            "wait for path to be /org/",
+            "wait for element .behat-link-pa-report-open to be visible",
```

Comment:
`            "wait for element .behat-link-pa-report-open to be visible",`

Is the id prefix supposed to be a `#` instead of a `.`?

Reply:
This link in question doesn't have an id attached to it and it felt a bit wrong to add an id just for the sake of a test so I settled on selecting the class of the link. It doesn't matter if there are multiple links on the page as the test isn't linked to a specific report so it seemed like an OK solution.

Reply:
Seems there was an issue after looking at the accessibility test in Circle.
I think pa11y is waiting for the URL to be `https://digideps.local/report/{id}/overview` and then it will carry out the actions. 

Maybe setting the URL to `https://digideps.local/` will avoid that error.

Reply:
Good spot! Weird though, its working OK locally...I'll SSH into the container and see what the screenshot looks like.